tests/negative/CMakeLists.txt

```diff
@@ -29,22 +31,28 @@ foreach(file ${TOIT_TESTS})
   file(RELATIVE_PATH relative ${TOIT_SDK_SOURCE_DIR} ${toit_file})
   string(REPLACE " " "__" test_name ${relative})
 
-  add_test(
-    NAME ${test_name}
-    COMMAND ${CMAKE_COMMAND}
-        -DTOITVM=$<TARGET_FILE:toit.run>
-        "-DTEST=${relative}"
-        "-DGOLD=${gold_file}"
-        "-DLIB_DIR=${lib_dir}"
-        "-DNORMALIZE_GOLD=${NORMALIZE_GOLD}"
-        "-DTEST_ROOT=${TOIT_SDK_SOURCE_DIR}"
-        "-DGIT_VERSION=${TOIT_GIT_VERSION}"
-        "-DTMP=${CMAKE_BINARY_DIR}/tmp"
-        "-DCMAKE_SYSTEM_NAME=${CMAKE_SYSTEM_NAME}"
-        -P "${CMAKE_CURRENT_SOURCE_DIR}/run.cmake"
-    WORKING_DIRECTORY ${TOIT_SDK_SOURCE_DIR}
-  )
-  set_tests_properties(${test_name} PROPERTIES TIMEOUT 40)
+  if(NOT "${test_name}" IN_LIST TOIT_SKIP_TESTS)
```

Comment:
continue for skipping?

Reply:
If I 'continue' then the update command below is not added. It's not completely clear whether it makes sense to skip the test and still have the update-command, but I felt it was cleaner.